content/docs/reference/routes/from.mdx

```diff
@@ -16,21 +17,62 @@ import TabItem from '@theme/TabItem';
 
 ## Summary
 
-**From** is the externally accessible URL for the proxied request.
+The **From** route is the externally accessible URL for a proxied HTTP request.
 
 Specifying `tcp+https` for the scheme enables [TCP proxying](/docs/capabilities/tcp/) support for the route. You may map more than one port through the same hostname by specifying a different `:port` in the URL.
 
 ## How to configure
 
-The `URL` must contain a **scheme** and **hostname** and cannot contain a path.
+The From URL must contain a **scheme** and **hostname**. It can't contain a path.
+
+### Port mapping behavior
+
+In v0.26, we've updated the default route matching behavior so that From routes defined _without_ a specific port in the route URL will match a request on any port.
+
+:::note
+
+You can disable this behavior with the [**Match Any Incoming Port**](/docs/reference/runtime-flags) runtime flag.
+
+:::
+
+With a From route like the one below:
+
+```yaml
+from: https://www.example.com
+```
+
+Pomerium will match incoming requests on any port:
+
+```yaml
+https://www.example.com
+
+https://www.example.com:443
+
+https://www.example.com:8443
+
+https://www.example.com:3000
+```
+
+If you specify the port in the From URL, Pomerium will only match incoming requests that specify this port:
+
+```yaml
+from: https://www.example.com:8443
+```
 
 :::note
 
-Only secure schemes (`https` and `tcp+https`) are supported.
+When defining a From route, you must use `https` or `tcp+https`. Pomerium only supports secure schemes.
```

Comment:
I'd suggest moving this note up, so it stays together with the sentence "The From URL must contain a scheme and hostname".

Or, it might make sense to remove the `:::note` callout and add this directly to that paragraph:

> The From URL must contain a **scheme** and **hostname**. It can't contain a path. The scheme must be `https` or `tcp+https`.